packages/opentelemetry-instrumentation-anthropic/opentelemetry/instrumentation/anthropic/__init__.py

```diff
@@ -362,20 +362,20 @@ def wrapper(wrapped, instance, args, kwargs):
 
 
 def _create_metrics(meter: Meter, name: str):
-    token_counter = meter.create_counter(
-        name=f"llm.{name}.tokens",
+    token_counter = meter.create_histogram(
+        name="gen_ai.client.token.usage",
         unit="token",
         description="Number of tokens used in prompt and completions",
     )
 
     choice_counter = meter.create_counter(
-        name=f"llm.{name}.choices",
```

Comment:
May I know what is the background for this? I was opening an issue at https://github.com/open-telemetry/semantic-conventions/issues/1079 , also want to distinguish client side and server side llm call. Thanks